pysilcam/tests/test_process.py

```diff
@@ -19,6 +22,64 @@
 print('ROOTPATH',ROOTPATH)
 print('MODEL_PATH',MODEL_PATH)
 
+@unittest.skipIf((ROOTPATH is None),
```

Comment:
I am wondering: Does having this check in place make it possible to accidentally merge something that doesn't run the tests? Imagine the following:

* I update the container, and accidentally remove the model path
* Tests are run, but since ROOTPATH = None, it will simply skip the tests
* GitHub Actions says that all is ✔️ , without mentioning the skippet tests unless I check the logs (unlikely!)

I get that you want to make it easy during development, but I wonder if it is better to be honest to the local developer and failing the tests because the testdata has not been configured.

Reply:
Yes. Very good catch. I thought about this a long time ago because the bamboo system was the same, then I forgot about it! I wonder if it is possible to configure the github action to insist that all tests pass and are not skipped.

The skipping tests has been very useful for deploying pysilcam on small computers for logging, where we are not developing code, but just want to check that the code runs and the install is done properly.

Reply:
Perhaps this one can help?

https://pypi.org/project/pytest-error-for-skips/

Reply:
I will create an issue for this.